frontend/packages/dev-console/src/components/EmptyState.tsx

```diff
@@ -71,6 +74,9 @@ const ODCEmptyState: React.FC<Props> = ({
   const addActionExtensions = useExtensions<AddAction>(
     isAddAction,
   ).filter(({ properties: { hide } }) => (hide ? hide() : true));
+
+  const guidedTourList = getGuidedToursWithStatus();
```

Comment:
Fetching of tour could easily be handled by `GuidedTourTile` component itself. If there are tours it will render the tile else just return `null`.

Reply:
I didn't care much about where the mock data is coming from but I agree with Rohit here that it makes more sense to have this in `GuidedTourTile`.